### PR TITLE
Feature/#2199 adding validation for address form

### PR DESCRIPTION
**Pull request by paslavskyi9roman**
The validation has been added for address pop-up at step 2 (Personal data)
зображення

Original task https://github.com/ita-social-projects/GreenCity/issues/2199

### CODE REVIEW

File: src/app/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.scss [hunk 1]
```diff
@@ -9,3 +9,7 @@ form {
 .btn:nth-child(2) {
   margin: 0 0 0 20px;
 }
+
+.validation {
+  color: #f00;
+}
```
Comment (yuriiLisunov) [inline src/app/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.scss:15]: use some var for this color

Comment (paslavskyi9roman) [reply src/app/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.scss]: variable for a color has been added with this commit ffc171f9490026bb04798d1380ae2744aa79012c

**Review by yuriiLisunov — APPROVED** [review]

File: src/app/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.ts [hunk 2]
```diff
@@ -36,36 +36,43 @@ export class UBSAddAddressPopUpComponent implements OnInit {
...
+      entranceNumber: ['', [
+        Validators.maxLength(2),
+        Validators.pattern(this.entranceNumberPattern)
+      ]],
+      longitude: ['', Validators.required],
+      latitude: ['', Validators.required]
     });
   }
 
+
```
Comment (JimHolman1199) [inline src/app/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.ts:75]: remove empty line 
change branch name from Feature/#-adding-validation-for-address-form to feature/#NUMBERHERE-adding-validation-for-address-form

Comment (paslavskyi9roman) [reply src/app/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.ts]: 1. Empty line has been removed + whole document has been formatted
2. The branch has been renamed

**Pull request closed without merging**